gui/options.cpp

```diff
@@ -2116,7 +2116,9 @@ void GlobalOptionsDialog::build() {
 	Common::String iconPath(ConfMan.get("iconspath", _domain));
 	Common::String extraPath(ConfMan.get("extrapath", _domain));
 
-	if (savePath.empty() || !ConfMan.hasKey("savepath", _domain)) {
+	if (ConfMan.hasKey("savepath", Common::ConfigManager::kSessionDomain)){
+		_savePath->setLabel(_("Command line temporary")); 
```

Comment:
I think it could be nice to still display the path, but maybe add a ` (temporary)` suffix to it.

Reply:
I agree it looks better this way